### PR TITLE
add kubeflow summit 2023

### PR DESCRIPTION
**Pull request by thesuperzapper**
This is the initial PR to add "Kubeflow Summit 2023" to the website.

NOTE: this is a first-pass so we can share the link, but we will make updates very soon.

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@akgraner @andreyvelich

**Comment by thesuperzapper** [comment]
@zijianjoy or @james-jwu, I would greatly appreciate a quick review on this, so we can share the links publicly.

File: config.toml [hunk 1]
```diff
@@ -36,6 +36,12 @@ ignoreFiles = [
 # Hugo - Top-level navigation (horizontal)
 ###############################################################################
 [menu]
+  [[menu.main]]
+    name = "Kubeflow Summit 2023 "
+    weight = -1000
+    pre = "<i class='fas fa-calendar pr-2' style='color: #FFC107'></i>"
+    post = "<br><span class='badge badge-warning'>Oct 6th</span> <span class='badge badge-warning'>Irving, Texas</span> <span class='badge badge-warning'>Virtual</span>"
```
Comment (andreyvelich) [inline config.toml:43]: Should we say Virtual/In-Person at the top so people can see it clearly ?
```suggestion
    post = "<br><span class='badge badge-warning'>Oct 6th</span> <span class='badge badge-warning'>Irving, Texas</span> <span class='badge badge-warning'>Virtual/In-Person</span>"
```

**Pull request merged**